doc/GetShellWithoutMonitorOnWindows.md

```diff
@@ -26,6 +26,11 @@
     ```
 1. If you will use any public or insecure networks, be sure to update your password using the instructions at the end of this document.
 1. Enter the additional SSID and Pre-Shared Key of the WiFi network and repeat as necessary.
+1. Now let's make sure ssh is enabled for our first boot (mandatory for Rasbian Buster or newer):
```

Comment:
This was/is also needed for Stretch, and even for the last few Jessie releases that came before Stretch.

Reply:
Sounds good. I'll change the wording.

Reply:
Upon further investigation, the empty ssh file should have been added (but wasn't) by setup-piForHeadlessConfig.ps1, despite non-error log output. So I'm actually going to close this one too and look into potentially making the PS scripts more resilient. Sorry for all the unnecessary traffic, and thank you for the feedback. I'll think through things a bit more before submitting anything else.